script/Common/00_00.txt

```diff
@@ -908,27 +912,27 @@ I heard a woman's voice, clear as a bell.
 -- Page 60, Offset 281.
 -- ただでさえちっこいんだから、草むらでうずくまってると見えないのよね。気をつけなさい、あやうく蹴り飛ばされるところだったんだから」
 You're small enough as it is. If you crouch down in the grass like that I can barely see you.
- You need to be more careful, I almost plowed right into you."
+ You need to be more careful, you nearly got sent flying."
 }
 [sha:6cf9af5c49cea474104c38bc611fd86e4b1a86b2]{
 -- Page 61, Offset 282.
 -- 女の人はふきげんそうに僕を指差した。
-The woman pointed at me grumpily.
+The woman points at me grumpily.
 }
 [sha:a85c6616e211d6d763bc9a13602c310bbdbea012]{
 -- Page 61, Offset 283.
 -- ……なんか、ちょっとあたまにきた。
-...I was a little pissed off.
+It's makes me a bit miffed...
```

Comment:
Might be a bit en_UK

Reply:
Seems fairly universal

Reply:
Ok, well also typo (it's makes). I'd also probably phrase it as
> I'm a bit miffed...

or 
> For some reason (なんか), that rubs me the wrong way...

Reply:
miffed is definitely a uk thing. nobody here says that

Reply:
Guess 

> For some reason, that rubs me the wrong way...

it is